lms/djangoapps/bulk_email/tasks.py

```diff
@@ -100,10 +83,6 @@
 # and should therefore not be retried.  For example, exceeding a quota for emails.
 # Also, any SMTP errors that are not explicitly enumerated above.
 BULK_EMAIL_FAILURE_ERRORS = (
-    SESAddressNotVerifiedError,  # Raised when a "Reply-To" address has not been validated in SES yet.
-    SESIdentityNotVerifiedError,  # Raised when an identity has not been verified in SES yet.
-    SESDomainNotConfirmedError,  # Raised when domain ownership is not confirmed for DKIM.
-    SESDailyQuotaExceededError,  # 24-hour allotment of outbound email has been exceeded.
```

Comment:
I see that we removed a number of these failure exceptions, are there any introduced in botocore/boto3 that we can/should take advantage of? 

I was looking to see if the ones we removed mapped to any of the new/currently supported exceptions (here: https://botocore.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#client-exceptions) and I saw items such as `AccountSendingPausedException`, `MailFromDomainNotVerifiedException`, `LimitExceededException` that seemed similar.

Reply:
Can you continue working on this? We have very limited domain knowledge here to replace all those exceptions.